Makefile

```diff
@@ -75,7 +75,7 @@ julia-libllvmcalltest: julia-deps
 julia-src-release julia-src-debug : julia-src-% : julia-deps julia_flisp.boot.inc.phony julia-cli-%
 	@$(MAKE) $(QUIET_MAKE) -C $(BUILDROOT)/src $*
 
-julia-cli-release julia-cli-debug: julia-cli-% :
+julia-cli-release julia-cli-debug: julia-cli-% : julia-deps
```

Comment:
I think this is not strictly necessary.  I agree it feels a little weird for the builder to get built before anything else, but it _is_ pretty stand-alone....

Reply:
It has happened pretty often for me on my new Nanosoldier builder

Reply:
FWIW, the conflict happens because of the CompilerSupportLibraries, which change the linking

Reply:
What happens?  I don't think anything in `cli` depends on anything in `deps`, does it?

Reply:
Ahhh, on Linux and such we may actually link against `libstdc++` and whatnot.  I think we are less standalone on Linux than we are on Windows.

How do you feel about that?  Would you prefer that we are more consistent and are equally standalone on all platforms?  This was not 100% purposeful on my part, it was more that because we have RPATH it's not fatal to depend on these compiler support libraries on Linux/macOS, but we can certainly make things more consistent.

Reply:
It doesn't really matter, we just need to declare it correctly here